docs/sla.md

```diff
@@ -0,0 +1,93 @@
+# Service Level Agreement(SLA)
+
+Service Level Agreement is available to Premium Customers only.
```

Comment:
Start the sentence with "This". 
What is the definition of "Premium Customer"? If defining in another document, should we refer to that other document?

Reply:
I agree with Burton about adding "The" to the start of the sentence. Also, if we define "Premium Customer" elsewhere, perhaps this reference should have a link that points users to that location.